### ISSUE TITLE
Stabilization

### ISSUE THREAD

**Issue by Kerbas-ad-astra** [WarrenSchultz/MonkeyWrench#4]
ModuleLiftingSurface only provides lift in one axis, so that's not going to work for the fins.  Probably the best option going forward is to scratch the fins from the adapter and make the airbrakes narrower so that fins and chutes can be placed between them.  It probably wouldn't be a bad idea to cut down the "bulkhead" of the cargo bay itself and bulk up the adapter, so that more space is available for attaching gizmos.

EDIT: Or, for that matter, make the pod just the 5m-wide barrel and make the adapter a 3.75-to-5m fuel tank, leaving it to the user to attach fins, parachutes, RCS, airbrakes, or whatever.

**Comment by WarrenSchultz** [comment]
Yeah, I was actually in the process of making that exact change (modularizing, as well as some of the other things you took care of with KIS etc) when the aero change hit, and I was waiting for things to shake out. Then the U5 upgrade got announced, so I'm waiting to see what that looks like. But I agree that it is the best way to go. Thanks for all your work to get things updated!
